hs-bindgen/src/HsBindgen/Backend/PP/Render.hs

```diff
@@ -173,7 +181,8 @@ prettyExpr env prec = \case
     EFree x  -> pretty x
     ECon n   -> pretty n
 
-    EIntegral i _ -> showToCtxDoc i -- TODO: why we have type annotation if we don't use it?
+    -- TODO: why we have type annotation if we don't use it?
```

Comment:
I keep TODO's on the line they referring to, so `git grep TODO` shows the context. Don't change.

Reply:
I see!  Thanks for letting me know.  I shall put this comment back on the same line.

I generally stay within the maximum line length (80 characters for this project), so I have likely moved such comments before when changing the code resulted in a line that exceeded the limit.  From now, I will remember this exception and try to keep such `TODO` on a relevant line in such cases.